com.unity.template-universal/ProjectSettings/QualitySettings.asset

```diff
@@ -93,7 +93,7 @@ QualitySettings:
     skinWeights: 255
     textureQuality: 0
     anisotropicTextures: 1
-    antiAliasing: 0
+    antiAliasing: 2
```

Comment:
Is this change intentional?

Reply:
my guess is yes, I didn't change any AA settings but my assumption here is that the Template hasn't had this file updated since URP started to set this depending on it's state, we still haven't separated the logic here right?